client/routes/search.js

```diff
@@ -45,6 +45,10 @@ module.exports = function (page) {
       var pageEl = document.getElementsByTagName('main')[0];
       pageEl.innerHTML = Templates['search'](ctx.state.data);
 
+      // Shows filter toggle button if javascript enabled
+      document.getElementById('fb').className = 'control__button';
+      document.querySelector('button.filterpanel__button').className = 'hidden';
```

Comment:
I understand you want to _overwrite_ here... why not in-line it instead of apply a class?
`document.querySelector('button.filterpanel__button').style.display = 'none';`